test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+source settings.sh
+
+help() {
+    cat <<-_EOF
+    
+test.sh
+This script creates virtual network adapter which can be used for bridging
+
+Usage:
+$0 [options]
+
+Options:
+    start   This option will create virtual adpater
+    stop    This option will destory virtual adpater
+    
+_EOF
+}
+
+if [ "$#" == 0 ]; then
+    echo "error: No arguments provided"
+    help
+    exit
+fi
+
+case "$1" in
+    "start" )
+        echo "creating virtual interface.."
+        sudo modprobe dummy
```

Comment:
You can sudo by running script with `sudo` for such case you need to check $UID before executing any command.